src/test/java/api/support/builders/FixedDueDateSchedule.java

```diff
@@ -20,51 +28,34 @@ public FixedDueDateSchedule(DateTime from, DateTime to, DateTime due) {
   }
 
   public static FixedDueDateSchedule wholeYear(int year) {
-    return dueAtEnd(
-      new DateTime(year, 1, 1, 0, 0, 0, DateTimeZone.UTC),
-      new DateTime(year, 12, 31, 23, 59, 59, DateTimeZone.UTC));
+    final LocalDate date = new LocalDate(year, 1, 1);
+
+    return dueAtEnd(atStartOfYear(date, UTC), atEndOfYear(date, UTC));
```

Comment:
Personally I might inline the local date.

```suggestion
    return dueAtEnd(atStartOfYear(new LocalDate(year, 1, 1), UTC), atEndOfYear(date, UTC));
```

Reply:
The same date variable is used twice there and not once.
```
return dueAtEnd(atStartOfYear(date, UTC), atEndOfYear(date, UTC));
````

Would instead become
```
return dueAtEnd(atStartOfYear(new LocalDate(year, 1, 1), UTC), atEndOfYear(new LocalDate(year, 1, 1), UTC));
```

Reply:
Ok, apologies, will close this.